test/Microsoft.IdentityModel.Protocols.Tests/HttpDocumentRetrieverTests.cs

```diff
@@ -179,6 +181,36 @@ public static TheoryData<DocumentRetrieverTheoryData> GetMetadataTheoryData
                     TestId = "Client Miss Configuration"
                 });
 
+                theoryData.Add(new DocumentRetrieverTheoryData
+                {
+                    Address = "https://login.windows.net/f686d426-8d16-42db-81b7-ab578e110ccd/.well-known/openid-configuration",
+                    DocumentRetriever = new HttpDocumentRetriever(HttpResponseMessageUtils.SetupHttpClientThatReturns("ValidJson.json", HttpStatusCode.RequestTimeout)),
+                    TestId = "RequestTimeout_RefreshSucceeds"
```

Comment:
Same test ID to the next test.

Reply:
Good catch. I've fixed the test name to reflect what's actually being tested. https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1784/commits/b7f4fa41af7dc0848266ade64d9284610c614ea6